src/macros.jl

```diff
@@ -1094,12 +1184,12 @@ end
 ## @orderby
 ##
 ##############################################################################
-
+# TODO: Determine which keyword arguments to expose to user
```

Comment:
TODO left

Reply:
Deleted the TODO, as it was unnecessary. 

We will think about keyword arguments for `@orderby` later.